locale/fi/LC_MESSAGES/django.po

```diff
@@ -759,3 +759,21 @@ msgstr "Voit lähettää palautetta osoitteeseen ltj.ymk@hel.fi"
 
 msgid "Log in again"
 msgstr "Kirjaudu uudelleen sisään"
+
+#, javascript-format
+msgid ""
+"This is the list of available %s. You may choose some by selecting them in "
+"the box below and then clicking the "Choose" arrow between the two boxes."
```

Comment:
Travis huomautti, että tämä antaa virheen: `keyword "Choose" unknown`.
Tämä johtuu siitä, että tässä translaation sisällä käytetään lainausmerkkejä, tämän voisi muuttaa esim. muotoon `"... clicking the 'Choose' arrow between..."`, eli sisemmät lainausmerkit muutettaisiin yksittäisiksi lainausmerkeiksi: " -> '

Reply:
Muutan yksittäisiksi lainausmerkeiksi.
Riveillä 78-84 on samalla lailla tuplalainausmerkit käytössä.